README.md

```diff
@@ -0,0 +1,13 @@
+# WMSE Scripts
+This repository contains miscellaneous scripts used by [Wikimedia Sverige (WMSE)](https://www.wikimedia.se). Most of these scripts are single file. 
+
+* fortnox_huvudbok_processing - *???*
```

Comment:
@lokal-profil, could you add a description for this?

Reply:
You can take the first line from the Readme inside the folder (and maybe mention that there is a sub-readme for instructions.